src/_vars.scss

```diff
@@ -15,10 +15,26 @@ $secondary-yellow: #fdb913;
 $secondary-orange: #de571f;
 $secondary-red: #b53228;
 
-$neutral-white: #fff;
+$neutral-white: #ffffff;
+$neutral-white-opacity50: #ffffff80;
+$neutral-white-opacity30: #ffffff4d;
 $neutral-light-gray: #ebeaea;
-$neutral-gray: #cccccb;
+$neutral-light-gray2: #dddddd;
+$neutral-gray: #808080;
+$neutral-gray2: #cccccb;
+$neutral-gray3: #adadad;
 $neutral-dark-gray: #31342b;
+$neutral-dark-gray2: #444444;
+$neutral-dark-gray3: #555555;
+
+$neutral-black: #000000;
```

Comment:
Thank you for updating this part, I had been wondering why there weren't more "global" theming variables for these commonly used style variations. I will make a note to incorporated these new theme options into the ApartApp code wherever it is appropriate.